### PR TITLE
python module leaks and fixes in UT

### PR DESCRIPTION
**Pull request by Kokan**
News file to be done.

### CODE REVIEW

**Comment by kira-syslogng** [comment]
Build SUCCESS

**Review by alltilla — APPROVED** [review]

File: modules/python/python-helpers.c [hunk 1]
```diff
@@ -211,6 +211,8 @@ _insert_to_dict(gpointer key, gpointer value, gpointer dict)
   PyObject *key_pyobj = _py_string_from_string((gchar *) key, -1);
   PyObject *value_pyobj = _py_string_from_string((gchar *) value, -1);
   PyDict_SetItem( (PyObject *) dict, key_pyobj, value_pyobj);
```
Comment (MrAnno) [inline modules/python/python-helpers.c:213]: >This function does not steal a reference to val.

**Review by MrAnno — APPROVED** [review]

**Comment by MrAnno** [comment]
Ohh, sorry. I did not realize the leak in the last commit is not in a unit test (I accidentally merged it without a news entry).

**Comment by Kokan** [comment]
Np, I'll open a new PR for that.

**Pull request merged**